<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" itemscope="" itemtype="http://schema.org/WebPage">
<head>
<meta http-equiv="content-type" content="text/html; charset=UTF-8" />
<meta http-equiv="X-UA-Compatible" content="chrome=1" />
<script type="text/javascript">/* Copyright 2008 Google. */ (function() { /*

Copyright The Closure Library Authors.
SPDX-License-Identifier: Apache-2.0
*/
(function(){function e(g){this.t={};this.tick=function(h,k,f){this.t[h]=[void 0!=f?f:(new Date).getTime(),k];if(void 0==f)try{window.console.timeStamp("CSI/"+h)}catch(m){}};this.getStartTickTime=function(){return this.t.start[0]};this.tick("start",null,g)}var a;if(window.performance)var d=(a=window.performance.timing)&&a.responseStart;var l=0<d?new e(d):new e;window.jstiming={Timer:e,load:l};if(a){var b=a.navigationStart;0<b&&d>=b&&(window.jstiming.srt=d-b)}if(a){var c=window.jstiming.load;0<b&&d>=
b&&(c.tick("_wtsrt",void 0,b),c.tick("wtsrt_","_wtsrt",d),c.tick("tbsd_","wtsrt_"))}try{a=null,window.chrome&&window.chrome.csi&&(a=Math.floor(window.chrome.csi().pageT),c&&0<b&&(c.tick("_tbnd",void 0,window.chrome.csi().startE),c.tick("tbnd_","_tbnd",b))),null==a&&window.gtbExternal&&(a=window.gtbExternal.pageT()),null==a&&window.external&&(a=window.external.pageT,c&&0<b&&(c.tick("_tbnd",void 0,window.external.startE),c.tick("tbnd_","_tbnd",b))),a&&(window.jstiming.pt=a)}catch(g){}})(); })()
</script>
<link rel="shortcut icon" type="image/x-icon" href="//www.google.com/images/icons/product/sites-16.ico" />
<link rel="apple-touch-icon" href="https://ssl.gstatic.com/sites/p/e44749/system/app/images/apple-touch-icon.png" type="image/png" />
<script type="text/javascript">/* Copyright 2008 Google. */ (function() { function d(a){return document.getElementById(a)}window.byId=d;function g(a){return a.replace(/^\s+|\s+$/g,"")}window.trim=g;var h=[],k=0;window.JOT_addListener=function(a,b,c){var f=new String(k++);a={eventName:a,handler:b,compId:c,key:f};h.push(a);return f};window.JOT_removeListenerByKey=function(a){for(var b=0;b<h.length;b++)if(h[b].key==a){h.splice(b,1);break}};window.JOT_removeAllListenersForName=function(a){for(var b=0;b<h.length;b++)h[b].eventName==a&&h.splice(b,1)};
window.JOT_postEvent=function(a,b,c){var f={eventName:a,eventSrc:b||{},payload:c||{}};if(window.JOT_fullyLoaded)for(b=h.length,c=0;c<b&&c<h.length;c++){var e=h[c];e&&e.eventName==a&&(f.listenerCompId=e.compId||"",(e="function"==typeof e.handler?e.handler:window[e.handler])&&e(f))}else window.JOT_delayedEvents.push({eventName:a,eventSrc:b,payload:c})};window.JOT_delayedEvents=[];window.JOT_fullyLoaded=!1;
window.JOT_formatRelativeToNow=function(a,b){a=((new Date).getTime()-a)/6E4;if(1440<=a||0>a)return null;var c=0;60<=a&&(a/=60,c=2);2<=a&&c++;return b?window.JOT_siteRelTimeStrs[c].replace("__duration__",Math.floor(a)):window.JOT_userRelTimeStrs[c].replace("__duration__",Math.floor(a))}; })()
</script>
<script>

  

  var breadcrumbs = [{"path":"../projects","deleted":false,"title":"Research","dir":"ltr"}];
  var JOT_clearDotPath = 'https://ssl.gstatic.com/sites/p/e44749/system/app/images/cleardot.gif';

  
  var JOT_userRelTimeStrs = ["a minute ago","__duration__ minutes ago","an hour ago","__duration__ hours ago"];

  
  

  

  var webspace = {"gvizGstaticVersion":"current","enableAnalytics":false,"pageSharingId":"jotspot_page","codeembeds":{"outerIframeSrc":"https://www.gstatic.com/jotspot/embeds/code/0f08d42392f2000e7e3f3daf5b427a43/outer_iframe.html","innerIframeSrc":"https://58885941-jotspot-embeds.googleusercontent.com/code/8d87fa64604b2a11fae2ed06104c58d3/inner_iframe.html"},"enableUniversalAnalytics":false,"sharingPolicy":"OPENED","siteTitle":"Anshul Kundaje","experiments":{"enableSubpagesGadgetInTakeout":true,"overrideDisableDomainEditing":false,"DisableSiteEditingFeature__disable_site_editing":true,"disableDomainEditing":true},"jot2atari":{"eligibility":"INELIGIBLE"},"onepickUrl":"https://docs.google.com/picker","adsensePublisherId":null,"features":{"moreMobileStyleImprovements":null,"subscriptionDataMigrationInProgress":null,"plusBadge":false},"configProperties":{"disableSiteEditing":null},"isPublic":true,"newSitesBaseUrl":"https://sites.google.com","isConsumer":true,"serverFlags":{"jot2AtariLearnMoreUrl":"https://support.google.com/sites/answer/7035197"},"domainAnalyticsAccountId":"","plusPageId":"","signInUrl":"https://accounts.google.com/AccountChooser?continue=../projects&service=jotspot","analyticsAccountId":"","scottyUrl":"/_/upload","homePath":"/","siteNoticeUrlEnabled":null,"plusPageUrl":"","adsensePromoClickedOrSiteIneligible":true,"csiReportUri":"https://gg.google.com/csi","sharingId":"jotspot","termsUrl":"//www.google.com/intl/en/policies/terms/","gvizVersion":1,"editorResources":{"sitelayout":["https://ssl.gstatic.com/sites/p/e44749/system/app/css/sitelayouteditor.css"],"text":["https://ssl.gstatic.com/sites/p/e44749/system/js/codemirror.js","https://ssl.gstatic.com/sites/p/e44749/system/app/css/codemirror_css.css","https://ssl.gstatic.com/sites/p/e44749/system/js/trog_edit__en.js","https://ssl.gstatic.com/sites/p/e44749/system/app/css/trogedit.css","../_/rsrc/1659597679000/system/app/css/editor.css","https://ssl.gstatic.com/sites/p/e44749/system/app/css/codeeditor.css","../_/rsrc/1659597679000/system/app/css/camelot/editor-jfk.css"]},"sharingUrlPrefix":"/_/sharing","isAdsenseEnabled":true,"domain":"defaultdomain","baseUri":"/site/anshulkundaje","name":"anshulkundaje","siteTemplateId":false,"siteNoticeRevision":null,"siteNoticeUrlAddress":null,"siteNoticeMessage":null,"page":{"isRtlLocale":false,"canDeleteWebspace":null,"isPageDraft":null,"parentPath":null,"parentWuid":null,"siteLocale":"en","timeZone":"America/Los_Angeles","type":"text","title":"Research","locale":"en","wuid":"wuid:gx:3c6d0695217364f3","revision":50,"path":"/projects","isSiteRtlLocale":false,"pageInheritsPermissions":null,"name":"projects","canChangePath":true,"state":"","properties":{},"bidiEnabled":false,"currentTemplate":{"path":"/system/app/pagetemplates/text","title":"Web Page"}},"canPublishScriptToAnyone":true,"user":{"keyboardShortcuts":true,"sessionIndex":"","guest_":true,"displayNameOrEmail":"guest","userName":"guest","uid":"","renderMobile":false,"domain":"","namespace":"","hasWriteAccess":false,"namespaceUser":false,"primaryEmail":"guest","hasAdminAccess":false,"isGoogleAdmin":false},"gadgets":{"baseUri":"../system/app/pages/gadgets"}};
  webspace.page.breadcrumbs = breadcrumbs;

  
  var JOT_siteRelTimeStrs = ["a minute ago","__duration__ minutes ago","an hour ago","__duration__ hours ago"];

</script>
<script type="text/javascript">
                window.jstiming.load.tick('scl');
              </script>
<meta name="title" content="Research - Anshul Kundaje" />
<meta itemprop="name" content="Research - Anshul Kundaje" />
<meta property="og:title" content="Research - Anshul Kundaje" />
<meta name="description" content="My homepage" />
<meta itemprop="description" content="My homepage" />
<meta id="meta-tag-description" property="og:description" content="My homepage" />
<style type="text/css">
</style>
<link rel="stylesheet" type="text/css" href="https://ssl.gstatic.com/sites/p/e44749/system/app/themes/wintermint/standard-css-wintermint-ltr-ltr.css" />
<link rel="stylesheet" type="text/css" href="../_/rsrc/1659597679000/system/app/css/overlay.css?cb=wintermint14a%25150goog-ws-left-noheadcontentthemedefaultcenter" />
<link rel="stylesheet" type="text/css" href="../_/rsrc/1659597679000/system/app/css/camelot/allthemes-view.css" />
<!--[if IE]>
          <link rel="stylesheet" type="text/css" href="../system/app/css/camelot/allthemes%2die.css" />
        <![endif]-->
<title>Research - Anshul Kundaje</title>
<meta itemprop="image" content="../_/rsrc/1472849833062/projects/KundajeLab_logo.png?height=300&amp;width=400" />
<meta property="og:image" content="../_/rsrc/1472849833062/projects/KundajeLab_logo.png?height=300&amp;width=400" />
<script type="text/javascript">
                window.jstiming.load.tick('cl');
              </script>
</head>
<body xmlns="http://www.google.com/ns/jotspot" id="body" class=" en            ">
<div id="sites-page-toolbar" class="sites-header-divider">
<div xmlns="http://www.w3.org/1999/xhtml" id="sites-status" class="sites-status" style="display:none;"><div id="sites-notice" class="sites-notice" role="status" aria-live="assertive"> </div></div>
</div>
<div id="sites-chrome-everything-scrollbar">
<div id="sites-chrome-everything" class="">
<div id="sites-chrome-page-wrapper" style="direction: ltr">
<div id="sites-chrome-page-wrapper-inside">
<div xmlns="http://www.w3.org/1999/xhtml" id="sites-chrome-header-wrapper" style="height:auto;">
<table id="sites-chrome-header" class="sites-layout-hbox-centered" cellspacing="0" style="height:auto;">
<tr class="sites-header-primary-row" id="sites-chrome-userheader">
</tr>
<tr class="sites-header-secondary-row" id="sites-chrome-horizontal-nav">
<td colspan="2" id="sites-chrome-header-horizontal-nav-container" role="navigation">
</td>
</tr>
</table>
</div>
<div id="sites-chrome-main-wrapper">
<div id="sites-chrome-main-wrapper-inside">
<table id="sites-chrome-main" class="sites-layout-hbox" cellspacing="0" cellpadding="{scmCellpadding}" border="0">
<tr>
<td id="sites-chrome-sidebar-left" class="sites-layout-sidebar-left initial" style="width:150px">
<div xmlns="http://www.w3.org/1999/xhtml" id="COMP_2bd" class="sites-embed" role="navigation"><h4 class="sites-embed-title">Main</h4><div class="sites-embed-content sites-sidebar-nav"><ul role="navigation" jotId="navList"><li class="nav-first "><div dir="ltr" style="padding-left: 5px;"><a href="../Home" jotId="wuid:gx:433ef46b7e972855" class="sites-navigation-link">Home</a></div></li><li class=""><div dir="ltr" style="padding-left: 5px;"><a href="../cv" jotId="wuid:gx:4d13fb8ec3c7f022" class="sites-navigation-link">C.V.</a></div></li><li class=""><div dir="ltr" style="padding-left: 5px;"><a href="../publication" jotId="wuid:gx:5f0fe9af8154b88d" class="sites-navigation-link">Publications</a></div></li><li class=""><div dir="ltr" style="padding-left: 5px;"><a href="../positions" jotId="wuid:gx:42f31b36692f965c" class="sites-navigation-link">Positions</a></div></li><li class=""><div dir="ltr" style="padding-left: 5px;"><a href="../contact" jotId="wuid:gx:4069f51dcc19a073" class="sites-navigation-link">Contact</a></div></li><li class=""><div dir="ltr" style="padding-left: 5px;"><a href="../system/app/pages/sitemap/hierarchy" jotId="wuid:gx:32fbb43a5c128fa2" class="sites-navigation-link">Sitemap</a></div></li></ul></div></div>
<div xmlns="http://www.w3.org/1999/xhtml" id="COMP_06257089060978926" class="sites-embed" role="navigation"><h4 class="sites-embed-title">Research</h4><div class="sites-embed-content sites-sidebar-nav"><ul role="navigation" jotId="navList"><li class="nav-first "><div dir="ltr" style="padding-left: 5px;"><a href="../students" jotId="wuid:gx:f843c1f5782c01e" class="sites-navigation-link">Lab Members</a></div></li><li class=""><div class="current-bg" jotId="wuid:gx:3c6d0695217364f3" dir="ltr" style="padding-left: 5px;">Research</div></li><li class=""><div dir="ltr" style="padding-left: 5px;"><a href="../teaching" jotId="wuid:gx:7e55b3f06dda3d0f" class="sites-navigation-link">Teaching</a></div></li><li class=""><div dir="ltr" style="padding-left: 5px;"><a href="../inotes" jotId="wuid:gx:28e13311049878b5" class="sites-navigation-link">Tutorials</a></div></li><li class=""><div dir="ltr" style="padding-left: 5px;"><a href="../idatasets" jotId="wuid:gx:570567f690f155d0" class="sites-navigation-link">Datasets</a></div></li><li class=""><div dir="ltr" style="padding-left: 5px;"><a href="../icode" jotId="wuid:gx:55d317d240e97d" class="sites-navigation-link">Code</a></div></li><li class=""><div dir="ltr" style="padding-left: 5px;"><a href="../fellowships" jotId="wuid:gx:596e89a980349a15" class="sites-navigation-link">Fellowships</a></div></li><li class=""><div dir="ltr" style="padding-left: 5px;"><a href="../conferences" jotId="wuid:gx:19ef6fa70fb85b6b" class="sites-navigation-link">Conferences</a></div></li></ul></div></div>
<div xmlns="http://www.w3.org/1999/xhtml" id="COMP_11533869912938033" class="sites-embed" role="navigation"><h4 class="sites-embed-title">Personal</h4><div class="sites-embed-content sites-sidebar-nav"><ul role="navigation" jotId="navList"><li class="nav-first "><div dir="ltr" style="padding-left: 5px;"><a href="../irecipes" jotId="wuid:gx:3cb6d31bc81059cd" class="sites-navigation-link">Recipes</a></div></li></ul></div></div>
</td>
<td id="sites-canvas-wrapper">
<div id="sites-canvas" role="main">
<div id="goog-ws-editor-toolbar-container"> </div>
<div xmlns="http://www.w3.org/1999/xhtml" id="title-crumbs" style="">
</div>
<h3 xmlns="http://www.w3.org/1999/xhtml" id="sites-page-title-header" style="" align="left">
<span id="sites-page-title" dir="ltr" tabindex="-1" style="outline: none">Research</span>
</h3>
<div id="sites-canvas-main" class="sites-canvas-main">
<div id="sites-canvas-main-content">
<table xmlns="http://www.w3.org/1999/xhtml" cellspacing="0" class="sites-layout-name-one-column sites-layout-hbox"><tbody><tr><td class="sites-layout-tile sites-tile-name-content-1"><div dir="ltr"><div><span style="background-color:transparent;font-size:10pt"><font face="arial, sans-serif"><br /></font></span></div><div><span style="background-color:transparent;font-size:10pt"><font face="arial, sans-serif">Click on the project listing below for more information about each.</font></span></div><div>
<div id="sites-toc-undefined" jotId="sites-toc-undefined" style="display:block; margin:5px auto 5px 0; text-align:left; width:auto;" class="sites-embed sites-sidebar-nav">
<h4 class="sites-embed-title">Projects</h4>
<div class="sites-embed-content" onclick="JOT_SUBPAGE_click(event)" style="text-align: left;">
<ul jotId="navList">
<li class="topLevel nav-first"><div dir="ltr" style="padding-left: 19px;"><a href="../projects/cagt" jotId="wuid:gx:6b42810c12da1000" class="sites-navigation-link topLevel">(2012) ENCODE: CAGT portal for Clustered Aggregation plots of functional marks at regulatory elements</a>
</div></li>
<li class="topLevel"><div dir="ltr" style="padding-left: 19px;"><a href="../projects/encode-data-quality" jotId="wuid:gx:70c940feb39e4de" class="sites-navigation-link topLevel">(2012) ENCODE: Comprehensive data quality measures and metadata</a>
</div></li>
<li class="topLevel parent" wuid="gx:30e6504e71e07e1a"><div dir="ltr" style="padding-left: 0px;"><div class="expander"></div><a href="../projects/idr" jotId="wuid:gx:30e6504e71e07e1a" class="sites-navigation-link topLevel">(2012) ENCODE: TF ChIP-seq peak calling using the Irreproducibility Discovery Rate (IDR) framework</a>
</div><ul>
<li class=""><div dir="ltr" style="padding-left: 38px;"><a href="../projects/idr/deprecated" jotId="wuid:gx:6e44073b064a7955" class="sites-navigation-link">Deprecated Pipeline (2012 version)</a>
</div></li>
</ul>
</li>
<li class="topLevel"><div dir="ltr" style="padding-left: 19px;"><a href="../projects/wiggler" jotId="wuid:gx:3f74277ea5965f18" class="sites-navigation-link topLevel">(2012) ENCODE: Wiggler - Normalized genome-wide signal coverage tracks of ENCODE data</a>
</div></li>
<li class="topLevel"><div dir="ltr" style="padding-left: 19px;"><a href="../projects/mappability" jotId="wuid:gx:4f9774575874597f" class="sites-navigation-link topLevel">(2012) Unique mappability tracks for several species</a>
</div></li>
<li class="topLevel"><div dir="ltr" style="padding-left: 19px;"><a href="../projects/chromatinvariationpilot" jotId="wuid:gx:642218f160ef17a1" class="sites-navigation-link topLevel">(2013) Chromatin State Variation across humans</a>
</div></li>
<li class="topLevel"><div dir="ltr" style="padding-left: 19px;"><a href="../projects/encodehistonemods" jotId="wuid:gx:306a1c492fc08e23" class="sites-navigation-link topLevel">(2014) ENCODE: Histone modification ChIP-seq uniform peak calls</a>
</div></li>
<li class="topLevel"><div dir="ltr" style="padding-left: 19px;"><a href="../projects/blacklists" jotId="wuid:gx:5659a9ecf43d54df" class="sites-navigation-link topLevel">(2014) mod/mouse/humanENCODE: Blacklisted genomic regions for functional genomics analysis</a>
</div></li>
<li class="topLevel"><div dir="ltr" style="padding-left: 19px;"><a href="../projects/modencode" jotId="wuid:gx:7face58c3f1b54bd" class="sites-navigation-link topLevel">(2014) mod/mouse/humanENCODE: Uniformly processed data (June 2012 Freeze)</a>
</div></li>
<li class="topLevel"><div dir="ltr" style="padding-left: 19px;"><a href="../projects/epigenomeroadmap" jotId="wuid:gx:108a43a0c2268f7d" class="sites-navigation-link topLevel">(2014) Roadmap Epigenomics Project: Chromatin States across 100s of human cell/tissue types</a>
</div></li>
<li class="topLevel"><div dir="ltr" style="padding-left: 19px;"><a href="../projects/videos" jotId="wuid:gx:59ef20e7015ec1ac" class="sites-navigation-link topLevel">Lecture and Talk videos</a>
</div></li>
</ul>
</div>
</div>
</div><div><hr /></div><h2><a name="TOC-Research-Summary"></a><font face="arial, sans-serif">Research Summary</font></h2><div><span style="color:rgb(34,34,34);font-size:10pt;font-family:arial,sans-serif;background-color:transparent"><div style="display:inline;float:left;margin:5px 10px 0px 0px"><a href="../projects/KundajeLab_logo.png?attredirects=0" imageanchor="1"><img border="0" height="300" src="../_/rsrc/1472849833062/projects/KundajeLab_logo.png?height=300&amp;width=400" width="400" /></a>      </div></span></div><div><h2><a name="TOC-Mission"></a><font face="arial, sans-serif">Mission</font></h2><font face="arial, sans-serif">Over the last decade, the maturation of high-throughput biotechnological platforms such as massively parallel sequencing has spurred a <b>data-driven</b> revolution in biomedical research. 2003 marked a major milestone when the Human Genome Project released the first consensus sequence of a complete human genome. Since then, there has been a tremendous acceleration in the field of <b>large-scale functional genomics</b> that uses high-throughput experimental techniques to measure the genome-wide activity of 1000s of cellular moieties with the goal of trying to decipher functions of different parts of the genomic sequence and unravel the dynamic control systems that regulate all cellular processes. Large, publicly-funded, coordinated consortia are generating massive compendia of heterogeneous functional genomic data. Now, more than ever, biologists are looking to engineering and computational techniques to help organize, analyze and interpret the deluge of data. </font><div><font face="arial, sans-serif"><i>My long-term research interest is the development of statistical and machine learning methods for integrative analysis of heterogeneous, high-throughput biological data to obtain a systems-level view of cellular mechanisms and complex diseases.</i></font><div><hr /><div><h2><a name="TOC-A-biological-primer"></a><font face="arial, sans-serif">A biological primer</font></h2><font face="arial, sans-serif">The genomic DNA sequence of a living organism encodes a myriad of functional elements. <b>Genes</b> are genomic elements whose DNA sequences are <b>transcribed</b> to intermediate <b>RNA</b> molecules that are further translated to <b>proteins</b>. Proteins perform a variety of critical cellular functions including the transcription of genes and the <b>regulation of gene expression</b> i.e. controlling the timing and amount of synthesis of RNA and proteins. A living cell adapts and responds to changes in its internal and external environment primarily by regulating gene expression. The regulatory mechanisms that control gene expression have <b>static and dynamic components</b>. All genes have their respective <b>regulatory domains</b> in the genome that are composed of a variety of short <b>DNA sequence motifs</b> to which specific <b>regulatory proteins</b> bind to activate, enhance or repress gene expression. This <b>combinatorial grammar</b> of regulatory sequence elements is statically encoded in the genome. However, the regulatory activity of these elements is also dynamically controlled by altering the synthesis of the regulatory proteins that bind these elements. There is another layer of dynamic control in the form of a plethora of chemical modifications that can be made to DNA and to structural proteins that the genomic DNA is wrapped around (collectively known as <b>chromatin</b>). These <b>epigenomic modifications</b> can completely override or complement the regulatory information encoded in the genomic sequence by affecting various properties of chromatin (e.g. compactness) thereby controlling the accessibility of regulatory elements by regulatory proteins. In a multi-cellular organism, the tremendous morphological and physiological diversity of different cell types which share the same genomic DNA sequence is also largely a consequence of dramatic differences in epigenomic activation or silencing of genomic domains. <b>Complex diseases</b> are often a consequence of <b>subtle disruptions to different combinations of functional elements</b>, which collectively disrupt complex <b>regulatory pathways</b>. Hence, deciphering the entire regulatory parts-list of the cell, understanding its static and dynamic components and unraveling the complex network of regulatory interactions that control gene expression are key steps to obtaining a systems-level understanding of cellular response and disease.</font></div><div><hr /><h2><a name="TOC-Past-current-and-future-research-goals"></a><font face="arial, sans-serif">Past, current and future research goals</font></h2><font face="arial, sans-serif">My past and current research spans three major areas each critical to understanding gene regulatory mechanisms. <br /></font><h3><a name="TOC-1.-A-comprehensive-functional-annotation-of-the-human-genome:"></a><font face="arial, sans-serif">1.      A comprehensive functional annotation of the human genome: </font></h3><font face="arial, sans-serif">Only about 1.5% of the human genome codes for genes. The remaining intergenic DNA contains a bewildering number of undiscovered and poorly understood regulatory elements. In order to reverse engineer the complex regulatory pathways that affect gene expression, we first need to obtain a comprehensive collection of regulatory elements. The <b>ENCODE (Encyclopedia of DNA elements)</b> project was launched in 2003 to pick up where the Human Genome Project left off, with this exact goal of systematically identifying all functional elements in the human genome [1]. I served as the lead data coordinator and one of the primary computational analysts for the ENCODE consortium from 2008 to 2012. I developed novel computational methods and collaborated extensively with experimental and computational research groups to reveal the regulatory architecture of the largest collection of human functional elements to date. I co-authored 10 publications in 2012 in the prestigious journals Nature, Genome Research and Genome Biology. Several independent scientific reviews rated ENCODE as one of the top research advances of 2012. My main research contributions were as follows: <br /><br /><b><u>Discovering regulatory elements by extracting biological signal from noisy experiments:</u></b> The ENCODE experimental groups generated over 2000 diverse functional genomic datasets using a variety of protocols, over a span of more than 4 years with rapidly evolving biotechnological platforms. These high-throughput experiments are riddled with various types of <b>noise, artifacts and systematic biases</b> and the first step to successful data integration is effective <b>noise filtering and normalization of data</b>. For example, <b>genome-wide binding maps of 100s of regulatory proteins</b> were generated in a large number of normal and cancerous human cell-lines. These binding maps consist of noisy, continuous, genome-wide signals roughly representing the probability of binding of specific regulatory proteins at each location in the genome. These raw data need to be processed in order to identify high-confidence binding sites as potential regulatory elements. To handle the immense scale and diversity of datasets, I developed a novel signal processing pipeline for robust, automated discovery of regulatory elements (collaboration with Prof. Qunhua Li, Penn State and Prof. Peter Bickel, UC-Berkeley). We leveraged a powerful, probabilistic framework (based on copula-mixture models) known as the Irreproducible Discovery Rate (IDR) that uses the rank-based reproducibility of binding signal across replicate datasets of an experiment in order to identify a critical threshold indicating a transition from signal to noise [1,2]. I used the IDR framework to uniformly process all the ENCODE datasets and obtain a comprehensive collection of high confidence regulatory elements. This method and other novel statistical quality control measures that I developed are key components of the <b>official ENCODE data standards</b> which serve as <b>state-of-the-art guidelines to the scientific community</b> for analyzing functional genomics experiments [1,3,4]. <br /><br /><b><u>Deciphering higher-order organization and functional heterogeneity of regulatory elements:</u></b> A regulatory protein can perform multiple functions by interacting with and co-binding DNA with different combinations of other regulatory proteins. The comprehensive maps of discovered regulatory elements provided the unique opportunity to explore their <b>higher-order organization in the genome</b> and the <b>functional heterogeneity of regulatory proteins</b>. I developed a novel supervised learning formulation based on rule-based ensembles and regularized regression that was able to sort through the combinatorial complexity of possible regulatory interactions and learn statistically significant (non-random) item-sets of co-binding events at an unprecedented level of detail [3]. I discovered a large number of novel pairwise and higher order interactions, several of which were validated. We showed extensive evidence that regulatory proteins could switch co-binding partners at different sets of regulatory domains within a single cell-type and across different cell-types. More importantly, these context-specific sets of regulatory proteins were found to be in close proximity to (and potentially regulate) distinct functional categories of genes. Epigenomic chromatin modifications also play a key role in defining functions of regulatory domains. ENCODE generated genome-wide signal maps of several key chromatin modifications which allowed me to answer two key questions – <b>Do all binding sites of a regulatory protein share a similar chromatin environment? What are the similarities and differences of the chromatin environment at binding sites of different regulatory proteins?</b> To tease out this functional heterogeneity, I developed a novel clustering algorithm to discern distinct patterns of chromatin modification signals in the vicinity of binding sites of 100s of regulatory proteins [5]. We revealed for the first time, extensive diversity and asymmetry (hidden directionality) of chromatin signal patterns. The functional significance of these patterns was evident by the fact that different classes of regulatory proteins (activators and repressors) showed different types of chromatin patterns. Also, regulatory elements showing distinct chromatin signal patterns were associated with differences in expression levels of nearby genes, in local sequence content and in the composition of co-binding regulatory proteins. <br /></font><h3><a name="TOC-2.-Epigenomic-regulation-in-diverse-human-cell-types"></a><font face="arial, sans-serif">2.      Epigenomic regulation in diverse human cell-types</font></h3><font face="arial, sans-serif">Hundreds of different chromatin modifications have been identified to date. Different combinations of modifications (<b>chromatin states</b>) have been found to define different types of functional domains (e.g. activating, repressed, transcribed) in the genome. However, only a sparse number of chromatin states have been well characterized so far. Deciphering biological meaningful sets of modifications from the enormous number of possibilities is a daunting task. Moreover, the chromatin states of genomic domains are modified during cellular differentiation giving rise to different cell-types and these states are often disrupted in different diseases. Hence, elucidating the <b>dynamics of chromatin states</b> across different cell-types is a critical step to understand epigenomic mechanisms of regulation. Since March 2012, I have been working with the <b>Roadmap Epigenomics consortium</b> that has generated the largest collection of genome-wide signal maps of dozens of chromatin modifications in 100s of human primary cell-types and tissues. I have trained multivariate hidden Markov models on these signal maps jointly across all the cell-types in order to learn a limited repertoire of hidden chromatin states and automatically segment the human genome into cell-type specific regions annotated with different chromatin state labels. These dynamic chromatin-state maps are not only revealing a staggering number of novel regulatory domains but are also allowing us to infer detailed similarities and differences of epigenomic regulation between the different cell-types. By correlating the dynamic transitions of chromatin state labels of regulatory elements with transcriptional activity of genes across cell-types, using novel probabilistic models, we have been able to <b>infer long-range regulatory interactions between distal regulatory elements and their target genes</b>. We are currently exploring new <b>large-scale learning</b> to reveal the <b>regulatory sequence grammars</b> underlying ~2.3 million dynamic regulatory elements discovered in the human genome.<br /></font><h3><a name="TOC-3.-Predictive-causal-unified-models-of-gene-regulation"></a><font face="arial, sans-serif">3.      Predictive, causal, unified models of gene regulation</font></h3><font face="arial, sans-serif">High-throughput experimental methods are allowing us to accurately measure gene expression levels of all genes in the genome in a variety of contexts such as different cell types or under various perturbations (e.g. drugs, stress and disease). From a systems standpoint, one can consider this global transcriptional response of the cell as the output of a <b>complex regulatory network</b>. Our goal is to reverse engineer the regulatory network (consisting of static and dynamic regulatory elements) by <b>linking regulatory element dynamics with gene expression dynamics</b>. I have developed a novel machine learning framework based on Boosting algorithms (an ensemble machine learning method) to learn predictive models of gene regulation from heterogeneous types of functional genomic features [6,7,8,9]. Each gene and every experimental context is represented by heterogeneous sets of millions of complementary and potentially interacting features such as DNA sequences of gene-specific regulatory domains, context-specific expression levels of regulatory proteins and activity profiles of diverse types of regulatory elements. Groups of genes that are co-regulated in specific groups of experimental contexts will tend to show correlated patterns of expression and also share similar sets of regulatory features. By jointly analyzing the dynamics of the transcriptional output of the regulatory system across a large number of contexts with these heterogeneous feature sets, the learning algorithm automatically selects a sparse set of informative features and also learns the statistical interactions between them to build a model (an ensemble of decision/regression trees) that can accurately and quantitatively predict context-specific gene expression. I have built novel learning modules (weak-learners) to automatically discover probabilistic representations of DNA sequence motifs from raw regulatory sequence data. I have used this framework to learn regulatory networks controlling stress responses in yeast and embryonic development in worms. I generated novel hypotheses about the activating or repressing roles of complex sub-networks of interacting regulatory elements, several of which were validated in follow up experiments [7].</font></div><div><font face="arial, sans-serif"><br /></font></div><div><span style="font-family:arial,sans-serif;font-size:10pt;background-color:transparent">Most regulatory network modeling approaches including my own have been very successful in simpler model organisms such as yeast and worm that have relatively simpler and better understood regulatory mechanisms, more complete functional annotations of their genomes and tremendous amounts of supporting data to learn from. Analogous efforts in humans have been less successful because the common tractable approach to handle the layered complexity of human regulation has been to analyze separate pieces of the regulatory system in isolation. ENCODE and other similar efforts have laid the foundations to build more complete and comprehensive models of regulation in humans. We have started extending some of my previous work on learning regulatory networks to model diverse regulatory mechanisms within </span><b style="font-family:arial,sans-serif;font-size:10pt;background-color:transparent">unified machine learning frameworks</b><span style="font-family:arial,sans-serif;font-size:10pt;background-color:transparent">. For example, in humans there are a large number of long-range regulatory connections between regulatory elements and target genes that are millions of bases (letters) apart on the linear genomic scale. The three dimensional folding structure of DNA brings these distal regulatory elements in physical proximity to their target genes. <b>Learning these long-range regulatory contact maps</b> will be extremely important to build predictive and accurate regulatory models. In order to effectively learn from vast and diverse feature spaces and massive amounts of data, significant improvements will be needed to the learning algorithms and model representations. So far batch learning algorithms dominate in computational genomics. We are exploring efficient online learning and other large-scale, distributed learning algorithms in order to rapidly update our models with the ever increasing amounts of data. </span><span style="font-family:arial,sans-serif;font-size:10pt;background-color:transparent">We are also exploring </span><b style="font-family:arial,sans-serif;font-size:10pt;background-color:transparent">deep learning architectures</b><span style="font-family:arial,sans-serif;font-size:10pt;background-color:transparent"> that are particularly good at capturing highly combinatorial functions by learning hierarchical feature representations that are well suited to capture the complexity of regulatory interactions. </span><span style="font-family:arial,sans-serif;font-size:10pt;background-color:transparent">In order to learn <b>causal statistical models</b>, we are developing machine learning methods to <b>integrate dynamic temporal and perturbation datasets specifically in the context of cellular differentiation (in skin) and reprogramming (using heterokaryons)</b>.</span></div><div><span style="font-family:arial,sans-serif;font-size:10pt;background-color:transparent"><br /></span></div><div><span style="font-family:arial,sans-serif;font-size:10pt;background-color:transparent">Finally, in computational biology, the </span><b style="font-family:arial,sans-serif;font-size:10pt;background-color:transparent">interpretability of learned models</b><span style="font-family:arial,sans-serif;font-size:10pt;background-color:transparent"> is of paramount importance. Learning sparse sets of informative features is especially difficult due to highly correlated, redundant features (an inherent property of biological systems conferring robustness) that are often equally important in terms of their biological function. I am keen to explore novel regularization and feature selection methods to overcome these problems. </span></div><div><h3><a name="TOC-4.-Regulatory-variation-in-personal-genomes-and-non-coding-genetic-variation-in-complex-diseases"></a><font face="arial, sans-serif">4.      Regulatory variation in personal genomes and non-coding genetic variation in complex diseases </font></h3><font face="arial, sans-serif">Homo sapiens comprise a single species. However, no two human beings are exactly identical due to <b>variations (mutations) in the personal genome</b> sequence of each individual. Alongside the functional genomics revolution, there have been large-scale efforts to systematically map the genetic diversity and variation of genomic sequence across diverse human populations and individuals. I intend to focus on two key questions that remain to be answered – <b>What is the variation of activity of regulatory elements across individuals and how does the underlying personal genetic variation affect regulatory mechanisms?</b> We have already begun work in the area of <b>cell-type specific regulatory variation</b> by exploring differences in chromatin state and gene expression in lymphoblastoid cell-lines from several individuals of diverse ancestry [10]. We are working on statistical and machine learning frameworks to uncover <b>complex regulatory mechanisms such as buffering, redundancy and compensation</b>. </font></div><div><font face="arial, sans-serif"><br /></font></div><div><font face="arial, sans-serif">Research efforts focused on human diseases are using massive case-control studies (known as genome-wide association studies) involving whole-genome sequencing of personal genomes to identify genetic sequence variants (mutations) that are significantly associated with disease. However, there is a lack of tools and methods to interpret the functional effects of these disease-associated variants, especially those lying outside protein-coding regions in the genome (known as <b>non-coding variants</b>). This is especially relevant since a majority of disease associated variants are non-coding. We are developing new computational methods to link disease-associated genetic variants with regulatory elements and decipher downstream deleterious effects on regulatory pathways and gene expression dynamics [11]. We are actively collaborating with several experts in various diseases and medical doctors to uncover genetic and regulatory mechanisms underlying <b>coronary heart disease, Alzheimer's disease, ageing and colorectal cancers</b>.<br /></font><hr /></div></div><div><h2><a name="TOC-Conclusion"></a><font face="arial, sans-serif">Conclusion</font></h2></div><div><span style="font-family:arial,sans-serif;font-size:10pt;background-color:transparent">Over the years, I have had the privilege of learning from and collaborating with a wide range of experts in genomics, computational biology and machine learning on some of the most exciting projects of this decade. In the process, I have developed a deep understanding of the underlying biology and mechanisms of gene regulation as well as the multitude of evolving biotechnological platforms. At the same time, I have significant expertise in formulating biological questions as computational problems and have developed a variety of statistical and machine learning approaches for massive integrative analysis of genomic data. We hope that our research efforts will open up new horizons in functional and disease genomics and ultimately improve human health!</span></div><hr /><h2><a name="TOC-References"></a><font face="arial, sans-serif">References</font></h2><font face="arial, sans-serif">1. Dunham I, Kundaje A, et al. ENCODE Project Consortium. An integrated Encyclopedia of DNA Elements in the human genome. Nature. 2012. PMID: 22955616<br />2.       Li Q et al. Measuring reproducibility of high-throughput experiments. The Annals of Applied Statistics. 2011 <br />3.       Gerstein MB, Kundaje A, et al. Architecture of the human regulatory network derived from ENCODE data. Nature. 2012. PMID: 22955619 <br />4.       Landt SG et al. ChIP-seq guidelines and practices of the ENCODE and modENCODE consortia. Genome Res. 2012. PMID: 22955991 <br />5.       Kundaje A et al. Ubiquitous heterogeneity and asymmetry of the chromatin environment at regulatory elements. Genome Res. 2012. PMID: 22955985 <br />6.       Middendorf M, Kundaje A, et al. Motif Discovery through predictive modeling of gene regulation, Proceeding of RECOMB, 2005 <br />7.       Kundaje A et al. Learning regulatory programs that accurately predict differential expression with MEDUSA. Ann NY Acad Sci. 2007. PMID: 17934055 <br />8.       Kundaje A et al. A predictive model of the oxygen and heme regulatory network in yeast. PLoS Comput Biol. 2008. PMID: 19008939 <br />9.       Lianoglou S, Kundaje A, et al. Inferring active signaling pathways by boosting the phosphorylation network. Proceeding of the NIPS Workshop on Computational Biology, 2006 </font></div><div><font face="arial, sans-serif">10. Kasowski M et al. Extensive variation of chromatin states across humans. Science, 2014<br />11.    Schaub MA, Boyle AP, Kundaje A, Batzoglou S, Snyder M. Linking disease associations with regulatory information in the human genome. Genome Res. 2012. PMID: 22955986</font><br /></div></div></div></td></tr></tbody></table>
</div> 
</div> 
<div id="sites-canvas-bottom-panel">
<div xmlns="http://www.w3.org/1999/xhtml" id="COMP_page-subpages" class="sites-canvas-bottom-panel-wrapper" style="">
<div class="sites-subpages">
            Subpages <span id="subpages-total-number">(11):</span>
<span>
<a href="../projects/cagt" dir="ltr">(2012) ENCODE: CAGT portal for Clustered Aggregation plots of functional marks at regulatory elements</a>
</span>
<span>
<a href="../projects/encode-data-quality" dir="ltr">(2012) ENCODE: Comprehensive data quality measures and metadata</a>
</span>
<span>
<a href="../projects/idr" dir="ltr">(2012) ENCODE: TF ChIP-seq peak calling using the Irreproducibility Discovery Rate (IDR) framework</a>
</span>
<span>
<a href="../projects/wiggler" dir="ltr">(2012) ENCODE: Wiggler - Normalized genome-wide signal coverage tracks of ENCODE data</a>
</span>
<span>
<a href="../projects/mappability" dir="ltr">(2012) Unique mappability tracks for several species</a>
</span>
<span>
<a href="../projects/chromatinvariationpilot" dir="ltr">(2013) Chromatin State Variation across humans</a>
</span>
<span>
<a href="../projects/encodehistonemods" dir="ltr">(2014) ENCODE: Histone modification ChIP-seq uniform peak calls</a>
</span>
<span>
<a href="../projects/blacklists" dir="ltr">(2014) mod/mouse/humanENCODE: Blacklisted genomic regions for functional genomics analysis</a>
</span>
<span>
<a href="../projects/modencode" dir="ltr">(2014) mod/mouse/humanENCODE: Uniformly processed data (June 2012 Freeze)</a>
</span>
<span>
<a href="../projects/epigenomeroadmap" dir="ltr">(2014) Roadmap Epigenomics Project: Chromatin States across 100s of human cell/tissue types</a>
</span>
<span>
<a href="../projects/videos" dir="ltr">Lecture and Talk videos</a>
</span>
</div>
</div>
<div id="sites-attachments-container">
</div>
<a xmlns="http://www.w3.org/1999/xhtml" name="page-comments"></a>
<div xmlns="http://www.w3.org/1999/xhtml" id="COMP_page-comments"><div class="sites-comment-docos-wrapper"><div class="sites-comment-docos"><div class="sites-comment-docos-background"></div><div class="sites-comment-docos-header"><div class="sites-comment-docos-header-title">Comments</div></div><div id="sites-comment-docos-pane" class="sites-comment-docos-pane"></div></div></div></div>
</div>
</div> 
</td> 
</tr>
</table> 
</div> 
</div> 
<div id="sites-chrome-footer-wrapper">
<div id="sites-chrome-footer-wrapper-inside">
<div id="sites-chrome-footer">
<div xmlns="http://www.w3.org/1999/xhtml" class="sites-subfooter"><div class="sites-subfooter-content"><div dir="ltr"><div style="display:block;text-align:center;margin-right:auto;margin-left:auto"><a href="../config/customLogo.gif?attredirects=0" imageanchor="1"></a>
<div style="text-align:center;display:block"><a href="../config/customLogo.gif?attredirects=0" imageanchor="1"></a>
<div style="display:block;text-align:left"><a href="http://cs.stanford.edu/sites/default/files/stanford_cs_logo.png" imageanchor="1" rel="nofollow"></a>
<div style="display:block;text-align:left"><font face="times new roman, serif" size="6">
<div style="display:block;text-align:left">
<div style="text-align:center;display:block">
<div style="display:block;text-align:left">
<div style="text-align:center;display:block"></div>
</div>
</div>
</div>
</font></div>
</div>
</div>
</div>
<div style="text-align:center"><span style="font-size:medium">
<div style="display:block;text-align:center;margin-right:auto;margin-left:auto"><a href="../config/customLogo.gif?attredirects=0" imageanchor="1"></a><div style="display:block;text-align:center;margin:5px auto 0px"><a href="../config/customLogo.gif?attredirects=0" imageanchor="1"></a><img border="0" src="../_/rsrc/1472849829035/config/customLogo.gif" />            <a href="http://cs.stanford.edu/sites/default/files/stanford_cs_logo.png" imageanchor="1" rel="nofollow"><img border="0" src="http://cs.stanford.edu/sites/default/files/stanford_cs_logo.png" /></a>       </div>
</div>
<br />
</span></div></div></div></div>
</div>
</div>
</div>
</div> 
</div> 
<div id="sites-chrome-adminfooter-container">
<div xmlns="http://www.w3.org/1999/xhtml" class="sites-adminfooter" role="navigation"><p><a class="sites-system-link" href="https://accounts.google.com/ServiceLogin?continue=../projects&amp;service=jotspot">Sign in</a><span aria-hidden="true">|</span><a class="sites-system-link" href="../system/app/pages/recentChanges">Recent Site Activity</a><span aria-hidden="true">|</span><a class="sites-system-link" href="../system/app/pages/reportAbuse" target="_blank">Report Abuse</a><span aria-hidden="true">|</span><a class="sites-system-link" href="javascript:;" onclick="window.open(webspace.printUrl)">Print Page</a><span aria-hidden="true">|</span><span class="sites-system-link">Powered By</span> <b class="powered-by"><a href="http://sites.google.com/site">Google Sites</a></b></p></div>
</div>
</div> 
</div> 
<script xmlns="http://www.w3.org/1999/xhtml" type="text/javascript">
    window.jstiming.load.tick('sjl');
  </script>
<script xmlns="http://www.w3.org/1999/xhtml" src="https://ssl.gstatic.com/sites/p/e44749/system/js/jot_min_view__en.js"></script>
<script xmlns="http://www.w3.org/1999/xhtml" type="text/javascript">
    window.jstiming.load.tick('jl');
  </script>
<script xmlns="http://www.w3.org/1999/xhtml" type="text/javascript">
      gsites.HoverPopupMenu.createSiteDropdownMenus('sites-header-nav-dropdown', false);
    </script>
<script xmlns="http://www.w3.org/1999/xhtml" type="text/javascript" defer="true">
            JOT_setupNav("2bd", "Main", false);
            JOT_addListener('titleChange', 'JOT_NAVIGATION_titleChange', 'COMP_2bd');
          </script>
<script xmlns="http://www.w3.org/1999/xhtml" type="text/javascript" defer="true">
            JOT_setupNav("06257089060978926", "Research", false);
            JOT_addListener('titleChange', 'JOT_NAVIGATION_titleChange', 'COMP_06257089060978926');
          </script>
<script xmlns="http://www.w3.org/1999/xhtml" type="text/javascript" defer="true">
            JOT_setupNav("11533869912938033", "Personal", false);
            JOT_addListener('titleChange', 'JOT_NAVIGATION_titleChange', 'COMP_11533869912938033');
          </script>
<script xmlns="http://www.w3.org/1999/xhtml" type="text/javascript">
              new sites.CommentPane('//docs.google.com/comments/d/AAHRpnXtASf_M2E6cjJ9Hgufts3RfH4dithXO5UQ9CVrQKktviyrZVYjkB0f-qZ1OW53fw847uMTvj1rEQMVqIiqq_OMKMUF705uzHG-sW57l0mk-dzANeFNha2--hnmWz-eRvipQteBM/api/js?anon=true',
                  false, false);
            </script>
<script xmlns="http://www.w3.org/1999/xhtml" type="text/javascript">
  setTimeout(function() {
    var fingerprint = gsites.date.TimeZone.getFingerprint([1109635200000, 1128902400000, 1130657000000, 1143333000000, 1143806400000, 1145000000000, 1146380000000, 1152489600000, 1159800000000, 1159500000000, 1162095000000, 1162075000000, 1162105500000]);
    gsites.Xhr.send('../_/tz', null, null, 'GET', null, null, { afjstz: fingerprint });
  }, 500);
</script>
<script xmlns="http://www.w3.org/1999/xhtml">
                    window.onload = function() {
                      if (false) {
                        JOT_setMobilePreview();
                      }
                      var loadTimer = window.jstiming.load;
                      loadTimer.tick("ol");
                      loadTimer["name"] = "load," + webspace.page.type + ",user_page";
                      window.jstiming.report(loadTimer, {}, 'https://gg.google.com/csi');
                    }
                  </script>
<script xmlns="http://www.w3.org/1999/xhtml" type="text/javascript">
        JOT_insertAnalyticsCode(false,
            false);
      </script>
<script xmlns="http://www.w3.org/1999/xhtml" type="text/javascript">
    var maestroRunner = new gsites.pages.view.SitesMaestroRunner(
        webspace, "en");
    maestroRunner.initListeners();
    maestroRunner.installEditRender();
  </script>
<script xmlns="http://www.w3.org/1999/xhtml" type="text/javascript" defer="true">
  //<![CDATA[
    // Decorate any fastUI buttons on the page with a class of 'goog-button'.
    if (webspace.user.hasWriteAccess) {
      JOT_decorateButtons();
    }

    // Fires delayed events.
    (function() {
      JOT_fullyLoaded = true;
      var delayedEvents = JOT_delayedEvents;
      for (var x = 0; x < delayedEvents.length; x++) {
        var event = delayedEvents[x];
        JOT_postEvent(event.eventName, event.eventSrc, event.payload);
      }
      JOT_delayedEvents = null;
      JOT_postEvent('pageLoaded');
    })();
  //]]>
</script>
<script xmlns="http://www.w3.org/1999/xhtml" type="text/javascript">
    JOT_postEvent('decorateGvizCharts');
  </script>
<script type="text/javascript">
          JOT_setupPostRenderingManager();
        </script>
<script type="text/javascript">
          JOT_postEvent('renderPlus', null, 'sites-chrome-main');
        </script>
<script type="text/javascript">
          sites.codeembed.init();
        </script>
<div id="server-timer-div" style="display:none"> </div>
<script type="text/javascript">
          window.jstiming.load.tick('render');
          JOT_postEvent('usercontentrendered', this);
        </script>
</body>
</html>
